editor/src/components/editor/action-types.ts

```diff
@@ -330,6 +330,7 @@ export interface ToggleCollapse {
 
 export interface AddToast {
   action: 'ADD_TOAST'
+  // FIXME: This contains React.ReactChild and is likely not serializable.
```

Comment:
is serializableness a requirement for actions? I think we have a bunch of actions that contain node modules and functions and other such stuff

Reply:
Not necessarily unless we want that for multiplayer, just happened to stumble on this as I worked on this.